sycl/include/sycl/ext/intel/experimental/esimd/emu/detail/esimd_emulator_device_interface.hpp

```diff
@@ -52,6 +52,7 @@ struct ESIMDDeviceInterface {
 
   ESIMDDeviceInterface();
 #include "esimd_emulator_functions_v1.h"
+#include "esimd_emulator_functions_v2.h"
```

Comment:
Shouldn't we change `ESIMD_DEVICE_INTERFACE_VERSION` to `2`?

Reply:
Interface version change is already applied in `sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp` : Line 127

Reply:
Right. Fixed it.